### DISCUSSION TITLE
why "git describe" does not see tags, produced by job?

### DISCUSSION THREAD

**Discussion by AndreyMlashkin** [stefanzweifel/git-auto-commit-action#332] [Help]
why "git describe" does not see tags, produced by job?

**Comment by stefanzweifel** [comment]
@AndreyMlashkin I currently don't have a live project where I use git-auto-commit to tag commits.
But this is the line in the code that creates the tag:
https://github.com/stefanzweifel/git-auto-commit-action/blob/7d0ca8f849305c56d36f992cfb1519b1af5b33f8/entrypoint.sh#L159

We use `-a` and `-m` to annotate the tag and add a message to it.

Do you maybe have a specific example of your problem/issue?
